scripts/tests/doppelganger_protection.sh

```diff
@@ -110,18 +119,27 @@ if [[ "$BEHAVIOR" == "success" ]]; then
     vc_4_keys_artifact_id="4-lighthouse-geth-$vc_4_range_start-$vc_4_range_end"
     service_name=vc-4
 
-    kurtosis service add \
-          --files /validator_keys:$vc_4_keys_artifact_id,/testnet:el_cl_genesis_data \
-          $ENCLAVE_NAME $service_name $LH_IMAGE_NAME -- lighthouse \
-          vc \
-          --debug-level debug \
-          --testnet-dir=/testnet \
-          --validators-dir=/validator_keys/keys \
-          --secrets-dir=/validator_keys/secrets \
-          --init-slashing-protection \
-          --beacon-nodes=http://$bn_2_url:$bn_2_port \
-          --enable-doppelganger-protection \
-          --suggested-fee-recipient 0x690B9A9E9aa1C9dB991C7721a92d351Db4FaC990
+    kurtosis service add $ENCLAVE_NAME $service_name --json-service-config - << EOF
+    {
+        "image": "$LH_IMAGE_NAME",
+        "files": {
+            "/validator_keys": ["$vc_4_keys_artifact_id"],
+            "/testnet": ["el_cl_genesis_data"]
+        },
+        "cmd": [
+            "lighthouse",
+            "vc",
+            "--debug-level", "info",
```

Comment:
Just noting this changed to info. That's probably fine for CI, as we will usually debug locally if there are issues.